app/src/main/java/com/work/restaurant/data/model/ExerciseSet.kt

```diff
@@ -0,0 +1,18 @@
+package com.work.restaurant.data.model
+
+import com.google.gson.annotations.SerializedName
+import com.work.restaurant.network.model.ExerciseSetResponse
+
+data class ExerciseSet(
+    @SerializedName("exerciseSet_name")
```

Comment:
```suggestion
    @SerializedName("exercise_set_name")
```
convention 에 맞게 변경이 필요해보여요
헌데 서버에서 받아오는 데이터가 아닌거로 아는데
@SerializedName 이 필요한가요??

Reply:
9b3ece7 반영하였습니다.